### PR TITLE
Dev wip1

### PR DESCRIPTION
**Pull request by l-arnold**
@DocCyblade How do we merge this manually?  Not clear on the conflicts

Seems aparent that editing in "github.com" with the "Default" being different than the branch we (at least I) commonly want to edit is a problem.

Might be best if "dev-wip1" become Default.  

Better yet, I start working more directly (SSH somehow).  Perhaps Github for Windows ?

### CODE REVIEW

File: conf.d/30_main [hunk 1]
```diff
@@ -34,13 +34,11 @@ alter user $DB_USER with encrypted password '$DB_PASS';
...
 # Set password in config file
-sed -i "s|db_password =.*|db_password = $DB_PASS|" $SERVER_CONF
+sed -i "s|db_password =.*|db_password = $DB_PASS|" $CONF
 
 # Stop postgresql server
 /etc/init.d/postgresql stop
-# 
-
-
+#
```
Comment (l-arnold) [inline conf.d/30_main]: conflicting?

File: conf.d/30_main [hunk 2]
```diff
@@ -108,7 +89,7 @@ a2enmod ssl proxy_http headers rewrite
 a2ensite openerp-ssl.conf		
 a2ensite odoo-80.conf		
 # WebConsole is disabled but available by typing the following line in shell
-a2ensite odoo-12325.conf		
+# a2ensite odoo-12325.conf
```
Comment (l-arnold) [inline conf.d/30_main:92]: wanting the comment "in place"  IE not called at build but available.

**Comment by l-arnold** [comment]
It appears that there were two Manual updates (trying to second trying to negate the first) on
--branch dev  conf.d/30-main
both of which be removed (and likely source of conflict)

first 5135a5f2b9e5f43efc6c830ae914b49af949aa52
second 5135a5f2b9e5f43efc6c830ae914b49af949aa52

trying to do this in Git Desktop...  Will see if it works.

**Comment by l-arnold** [comment]
Worked through using GIT Desktop.  Relatively helpful. ...

But having "build errors" in DEV.  
Seems like "make clean" is not as long of a "cleanup" as in dev-wip1.  May need to clear the folder and start fresh on this one.  Trying dev-wip1  (need to be traveling about now)

This is the command line last lines:
fab-chroot build/root.patched --script /turnkey/fab/common/conf/pgsql
- set postgres
- ls /etc/postgresql/
- PGSQL_VER=
- CONF_DIR=/etc/postgresql//main
- pg_dropcluster --stop main
  Usage: /usr/bin/pg_dropcluster [--stop] <version> <cluster>
  /usr/share/fab/product.mk:476: recipe for target 'build/stamps/root.patched' failed
  make: **\* [build/stamps/root.patched] Error 1

**Comment by JedMeister** [comment]
Looks like something is going wrong but it's nothing to do with Odoo build code.

`fab-chroot build/root.patched --script /turnkey/fab/common/conf/pgsql`
means that it was applying the conf script `pgsq` from common when it errored. So unless you've been editing files in common there is something else going on. Do you have plenty of free space on your TKLDev maching? Or was this posted before you discovered your free space issue?

**Comment by l-arnold** [comment]
This is after thebfree space issue.

I think was related to my very curiously typing something like chroot or deck...  Then restarting. Killing the foder etc.  Seems related to some ghost in tkldev around that one branch.

That said, I am.an optimist.

**Comment by JedMeister** [comment]
My guess is Postgres still running somewhere...

**Pull request merged**